packages/database/src/TupaiaDatabase.js

```diff
@@ -555,6 +560,19 @@ function buildQuery(connection, queryConfig, where = {}, options = {}) {
   return query;
 }
 
+function sanitizeComparisonValue(comparator, comparisonValue) {
+  switch (comparator) {
+    case COMPARATORS.LIKE:
+    case COMPARATORS.ILIKE: {
+      // escape underscores, as they are treated as wildcards in postgres which we generally don't want
+      return comparisonValue.replaceAll('_', '\\_');
```

Comment:
```suggestion
      return `%${comparisonValue.replaceAll('_', '\\_'))%`;
```

Can we also add `%` to match any value contains `comparisonValue`? 
```
'abc' LIKE '%c%'     true
```

Reply:
Thanks for the review @billli0! On your suggestion, I think that might be trickier, as we already explicitly set the `%` in many places throughout the code. Also, I think we probably want to be able to control the ability for matching anywhere in the string vs just at the start.

But I'm happy to change it to `%${value}%` in the Admin Panel 👍 I think it would be handy